index.html

```diff
@@ -17,12 +17,14 @@
       <span id="seconds">00</span>
     </div>
     <div id="buttons-container">
-      <button>Start</button>
-      <button>Pause</button>
-      <button>Stop</button>
+      <button type="button" class="btn btn-start">Start</button>
+      <button type="button" class="btn btn-pause">Pause</button>
+      <button type="button" class="btn btn-atop">Stop</button>
     </div>
   </div>
 
+  <script src="./jQuery.js"></script>
+  <script src="./popper.js"></script>
```

Comment:
We don't have these files locally so you can't refer to them. But you CAN refer to the CDN versions which are absolute links. Look at the documentation again and follow it closely. You're also missing one more JS file.